project/Dependencies.scala

```diff
@@ -2,6 +2,7 @@ import sbt._
 
 object Dependencies {
   val akkaV = "2.4.1"
+  val akkaStreamV = "2.4.17"
```

Comment:
Upgrading akka to the latest (turns out it's 24.19 since I last checked) works fine. 2.5 is a somewhat non-trivial change (see http://doc.akka.io/docs/akka/snapshot/scala/project/migration-guide-2.4.x-2.5.x.html) and definitely does not work without code changes ... I tried.